blocks/rich-text/index.js

```diff
@@ -149,6 +151,7 @@ export class RichText extends Component {
 		editor.on( 'BeforeExecCommand', this.maybePropagateUndo );
```

Comment:
Did we test to see how TinyMCE's undo history interacts with our own?

- Are we calling `onChange` with the correct value after TinyMCE finishes undo?
- Should we be bypassing TinyMCE's undo altogether (also preventing it from accumulating any history)?

Reply:
@aduth I'm proposing these changes in #4956. Good point about preventing TinyMCE accumulating any history at all. We just need the signals, not the store.